### DISCUSSION TITLE
Remove Network Status & Network Statistics from the app

### DISCUSSION THREAD

**Discussion by rafaeldjpbrochado** [iotaledger/firefly#1532] [Features / Enhancements]
I'd like to propose that we remove the following from the app:

![netstatus](https://user-images.githubusercontent.com/39466455/111080141-7886d400-84d3-11eb-9732-241db81b6813.png)

## Motivation

As a DLT geek, I love seeing the "network status" information. However, I recognize that for most people, this can lead to lots of questions and confusion. The average user doesn't even grasp how traditional servers work, let alone distributed ledgers. When I access Revolut, Paypal, Facebook or Messenger, I don't see a "server status" indicator anywhere. It's unnecessary. The ultimate goal of any network like ours is to stay online all the time. This applies to Paypal, Revolut, but also Firefly and IOTA. So by adding a "network status" indicator, we're subconsciously telling the user that we think there will be times when our network won't be live, and therefore they might find that indicator useful. I suggest we ditch it.

## Descriptions of possible user confusion

![netstatus2](https://user-images.githubusercontent.com/39466455/111080299-327e4000-84d4-11eb-9b8d-d8644abb4310.png)

## Wallet version

v0.1.0

**Comment by cvarley100** [comment]
@digital-zen another one for your thoughts

**Comment by rafaeldjpbrochado** [comment]
Node operators are generally advanced users, and even they sort of panic when they see a low percentage of confirmation rate (which according to muXxer, is not important at all). Have a look at this discussion below:

![chrystalistestnet](https://user-images.githubusercontent.com/39466455/111241744-2083c680-85d4-11eb-819b-d92a31db22ba.png)

I think we don't need to expose Firefly users to this kind of stuff.

**Comment by digital-zen** [comment]
Agree with a lot of the sentiment behind this. Propose we think of a solution to inform the user if the network is degraded in some way. 

This is based around the following presumptions 

- As a user, if everything is working as normal, I do not need to know.
- As a user, if the software is working sub-optimally and is impacting my experience, I should know what and why.

We can also consider showing this in dev mode only and/or having an option in settings to display, with off as a default outside of dev mode.

**Comment by plabbett** [comment]
I would like this information to be available to those who want it  (opt-in via the dev mode/advanced settings options are fine!) and possibly even expand what's provided. 

## Reason:

I can't find a way to quickly see the server I'm currently connected to, which is important to me when I'm using Manual node selection. Currently, I can only assume I'm connected to my own node by making it the only available node in the list (Primary setting only tells me preference, not current status.) 

If there is a way to see/find this information currently and I'm missing it then please share! 

But again just want to put my vote in for keeping this information in some form.

**Comment by rafaeldjpbrochado** [comment]
I liked the suggestion of having the indicator off by default but then available in the settings for those who do want to have it enabled at all times.

Additionally, as @digital-zen suggested, if the network is down or performing sub-optimally, we should notify the users (even if they have the network status indicator disabled).

**Comment by cvarley100** [comment]
We could disable it by default and enable in settings or in dev mode. A minimised form should be displayed to regular users i.e. Tangle not operational, internet disconnected etc.
